torch/ao/quantization/utils.py

```diff
@@ -245,10 +245,15 @@ def calculate_qmin_qmax(quant_min: int, quant_max: int, has_customized_qrange: b
             )
```

Comment:
should we get `initial_quant_min` and `initial_quant_max` based on dtype? currently it is hardcoded to 0, 255